doc/Product-requirements.md

```diff
@@ -0,0 +1,29 @@
+# Product requirements
+## Stakeholders
+- **Student** - Child at the age between 3 - 6, who experiences autism disorders. He or she is used to follow instructions from activity schedules prepared on purpose as a part of treatment. Every student has a different set of instructions depends on varied conditions e.g. age, level of disorder, previous experience or part of a treatment. Schedules are used in school and later in home.
+- **Therapist** - Teacher, who takes care of student's treatment. He or she prepares activity schedules dedicated for each student. Every teacher can look after more then one student.
+- **Parent**- Person, who works with the child at home with therapist's indications. He or she can use activity schedules prepared by teacher or on his/her own.
+
+## Main functionalities
+
+###Therapist use case diagram
+![Therapist use case diagram](therapist-use-cases.jpg)
+
+- As therapist I need to create a plan containg specified tasks for every of children to reach treatment goals.
+- As therapist I want to create tasks that I could assign to many plans and to one plan many times because I use different combination of available tasks during treatment.
+- As therapist I want to prepare a task that can have image or video, sound and/or text for children to better understand the issue.
+- As therapist I teach children with varied level of skills, so I need to be able to set different parameters like size of fonts, size of picture, type of displaying tasks and steps for each student.
+- As therapist I would like to limit execution of some tasks and step by timer, because some of them shouldn't take to much time.
+- As therapist I would like to switch between plans easly during its execution and remember where we have finished last time, because children are working with many plans during one session.
```

Comment:
You're right. Good point. We didn't asked about resetting progress, I think. @mikolevy do you remember anything about this?